pkg/signing/helpers.go

```diff
@@ -0,0 +1,13 @@
+package signing
+
+var _ Signer = &TestSigningProvider{}
+
+type TestSigningProvider struct {
+	PluginAdapter
+	//TODO: add a test signer here
```

Comment:
Good catch, I missed this TODO.
I would like to have one, but I'm not sure it can be done in a way that makes sense. I will spend some time thinking about it.